src/pudl/metadata/resources/sec10k.py

```diff
@@ -4,11 +4,14 @@
 
 RESOURCE_METADATA: dict[str, dict[str, Any]] = {
     "core_sec10k__assn_exhibit_21_subsidiaries_and_filers": {
-        "description": """A table associating subsidiaries listed in Exhibit 21 with
-their SEC central index key, if the subsidiary also files Form 10-K.
-
-Exhibit 21 subsidiaries and SEC 10-K filers are considered matched if they have
-identical names and the same location of incorporation.""",
+        "description": {
+            "additional_summary_text": (
+                "subsidiaries listed in "
+                "Exhibit 21 and their SEC central index key, if the subsidiary also files Form 10-K."
+            ),
+            "additional_details_text": "Exhibit 21 subsidiaries and SEC 10-K filers are "
+            "considered matched if they have identical names and the same location of incorporation.",
+        },
```

Comment:
it would be good to add a usage warning noting that the coverage of this assn is probably not 100% (its kind of implied but you have to read between the lines rn)

Reply:
we just chatted and decided to add more usage warnings in the context of https://github.com/catalyst-cooperative/pudl/issues/4347 or any follow up on the metadata migration